work/work.go

```diff
@@ -36,19 +40,35 @@ func ExpandPaths(origWD string, paths []string) ([]string, error) {
 			if strings.HasPrefix(path, ":") {
 				path = "."
 			}
-			// Join the path with the original working directory. We would have cd'ed to the root of the plz repo by this
-			// point
-			path = filepath.Join(origWD, path)
+		}
+
+		isWildcard := false
+		if filepath.Base(path) == "..." {
+			isWildcard = true
+			path = filepath.Dir(path)
 		}
 
 		path = filepath.Clean(path)
+		if filepath.IsAbs(path) {
+			p, err := filepath.Rel(wd, path)
+			if err != nil {
+				return nil, err
+			}
+			path = p
+		} else {
+			path = filepath.Join(origWD, path)
+		}
```

Comment:
I find this a little confusing... its not obvious what wd and origWD is and I don't really understand why we take relative path wrt wd if absoulte but then join origWD if not absolute. I feel a refactor or rename of variables might help

Reply:
So `origWD` is the original working directory. We cd to the repo root to avoid having to think about any of this. The rest of puku can assume that its working directory is the repo root. This is trying to make all the paths the user passed in be relative to the repo root.